### PR TITLE
allow users of hydrolysis-analyzer.html to specify the loader

### PR DESCRIPTION
**Pull request by garlicnation**

### CODE REVIEW

File: hydrolysis-analyzer.html [hunk 1]
```diff
@@ -65,6 +65,10 @@
         notify:   true,
       },
 
+      permissive: {
```
Comment (justinfagnani) [inline hydrolysis-analyzer.html]: what about a `resolver` property that defaults to `'xhr'`?

Comment (garlicnation) [inline hydrolysis-analyzer.html]: fsResolver is never correct for this usecase. I'd rather not let someone specify it.

Comment (justinfagnani) [inline hydrolysis-analyzer.html]: It is in Electron, where Designer runs :)

**Comment by garlicnation** [comment]
@justinfagnani PTAL

**Comment by justinfagnani** [comment]
What does "permissive loader" mean here. The PR title and change don't connect for me.

**Comment by garlicnation** [comment]
xhr loader, fs loader, and a permissive version of the loader that doesn't throw when it fails to load

**Comment by justinfagnani** [comment]
LGTM

fwiw, it's the word "permissive" in the title that confused me. I looked for something to do with a permissive loader in the change. This change just let's the user specify the loader in general, and thus the very general change makes sense.

In the future could resolver accept an loader instance rather than a string?

**Comment by garlicnation** [comment]
It could accept an instance, but that would require an API change to hydrolysis.Analyzer.analyze

**Comment by justinfagnani** [comment]
LGTM

fwiw, it's the word "permissive" in the title that confused me. I looked for something to do with a permissive loader in the change. This change just let's the user specify the loader in general, and thus the very general change makes sense.

In the future could resolver accept an loader instance rather than a string?

**Pull request merged**